packages/admin/admin/src/form/FieldContainer.tsx

```diff
@@ -251,13 +296,27 @@ export const FieldContainer = (inProps: PropsWithChildren<FieldContainerProps>)
                 )}
                 <InputContainer ownerState={ownerState} {...slotProps?.inputContainer}>
                     {children}
-                    {hasError && (
-                        <Error error {...slotProps?.error}>
-                            {error}
-                        </Error>
-                    )}
-                    {hasWarning && !hasError && <Warning {...slotProps?.warning}>{warning}</Warning>}
-                    {helperText && !hasError && !hasWarning && <HelperText {...slotProps?.helperText}>{helperText}</HelperText>}
+                    <HelperTextWrapper {...slotProps?.helperTextWrapper}>
+                        {hasError && (
+                            <Error error {...slotProps?.error}>
+                                {helperTextIcon && helperTextIcon}
```

Comment:
Please only use the `&&` syntax when checking a boolean, to prevent potentially rendering a falsy value. 
_(Same for the icons rendered below)_

```suggestion
                                {Boolean(helperTextIcon) && helperTextIcon}
```